src/components/timeline/timeline-event/timeline-event.component.html

```diff
@@ -0,0 +1,15 @@
+<li class="timeline-inverted">
+    <div class="timeline-badge {{ badgeColor }}">

```

Comment:
Use ngClass to set the class name rather than interpolation as it is specifically designed for handling the adding and removing of classes

Reply:
Using ngClass instead of interpolation to add a class.